iot_config/src/gateway_info.rs

```diff
@@ -143,13 +144,16 @@ pub(crate) mod db {
         db: impl PgExecutor<'_>,
         address: &PublicKeyBinary,
     ) -> anyhow::Result<Option<IotMetadata>> {
+        let entity_key = bs58::decode(address.to_string()).into_vec()?;
```

Comment:
are you sure this gives you what you need ? seems odd that the a public key in a format not directly supported by crypto-rs is being used here

Reply:
I’d like to go through it with folks today but I’ve tested this locally with a dump of data and this is unfortunately what we need ☹️